torch/csrc/distributed/rpc/rref_impl.h

```diff
@@ -263,6 +260,15 @@ class TORCH_API OwnerRRef final : public RRef {
   OwnerRRef& operator=(const OwnerRRef& other) = delete;
   OwnerRRef& operator=(OwnerRRef&& other) = delete;
 
+  OwnerRRef(worker_id_t ownerId, const RRefId& rrefId, TypePtr type)
```

Comment:
IIUC, this is to make sure that we can do sth like:

```c++
c10::make_intrusive<OwnerRRef>(getWorkerId(), rrefId, type);
```

The concern here is that we expose `OwnerRRef` as `TORCH_API`, so that C++ applications can create `OwnerRRef` on their own, which can mess up with the `RRefContext`. Is it possible to avoid exposing this by passing a pointer to `intrusive_ptr<OwnerRRef>` construction API?

Reply:
tried `intrusive_ptr<OwnerRRef>(new OwnerRRef())` way, and it does not seem to work because of intrusive_ptr limitation. I will create a issue to track this.

Reply:
created https://github.com/pytorch/pytorch/issues/33246